spyder/plugins/outlineexplorer/tests/test_widgets.py

```diff
@@ -267,7 +267,7 @@ def test_go_to_cursor_position_with_new_file(create_outlineexplorer, qtbot):
     # Click on the 'Go to cursor position' button of the outline explorer's
     # toolbar :
     filename = CASES['text']['file']
-    assert outlineexplorer.treewidget.currentItem() is None
```

Comment:
`OutlineExplorerToolbuttons.GoToCursor` is called during creation apparently. Unrelated to this PR I think.